pkg/scan/local/service_test.go

```diff
@@ -365,6 +365,13 @@ func TestScanner_Scan(t *testing.T) {
 					PkgRelationships: ftypes.Relationships,
 					Scanners:         types.Scanners{types.LicenseScanner},
 					LicenseFull:      true,
+					LicenseCategories: map[ftypes.LicenseCategory][]string{
+						ftypes.CategoryNotice: {
+							"MIT",
+							"text://(c) 2015 Continuum Analytics, Inc.",
+							"CUSTOM License: (c) 2016 Continuum...",
```

Comment:
Since we do exact matching, users have to exactly write `CUSTOM License: (c) 2016 Continuum...`. Am I correct? If so, it seems challenging for users to specify the license text in the config file.

Reply:
My thoughts were as follows:
1. user scans target
2. gets license in table report with text license. e.g. 
    ```
    ┌───────────────────────┬───────────────────────────────────────────┬────────────────┬──────────┐
    │        Package        │                  License                  │ Classification │ Severity │
    ├───────────────────────┼───────────────────────────────────────────┼────────────────┼──────────┤
    │ libpython3.10-minimal │ CUSTOM License: Redistribution and use... │ unknown        │ UNKNOWN  │
    ```
3. Just copies license from table report and fills into config file.

In #8847 the user uses `CUSTOM *  ` license ( (c) `In our case, all the GFDL-* and "CUSTOM License: * Permission to..." licenses should be categorised as restricted.`)

Reply:
Since the text is truncated in this table view due to the limited space, it may look like it’s only abbreviated visually. As a result, users might assume they need to specify the full text. However, in reality, they need to copy the ellipsis (...) exactly as shown. Is that correct? I might be missing something here. But if I’m right, specifying the text via exact match could be quite difficult, so perhaps we should consider supporting glob patterns.

Also, it might be a good idea to separate the PR for handling license texts.

Reply:
I think it's rare that a user creates licenses/categories themselves, so maybe we're worrying about it for nothing.
But I agree with you - let's split the logic into 2 PRs and think about text licenses separately.

I also like your idea about glob patterns.

Reply:
Created #8888 for text licenses